integration/suite.rb

```diff
@@ -72,11 +72,7 @@ def pink
       puts "#{test_name} #{'SUCCEEDED'.green}"
     ensure
       if options[:vm_maintenance] && succeeded
-        puts "Halting VM".yellow
-        unless system("vagrant halt")
-          puts "Tried to halt #{test_name} but it failed".red
-        end
-        puts "Halted VM"
+        
```

Comment:
The blank line seems conspicuous. Is something missing?  Is there another way to clean up the VM?